container-terminal.js

```diff
@@ -248,7 +247,7 @@
                             /* There's no term.hideCursor() function */
                             if (term) {
                                 term.cursorHidden = true;
-                                term.refresh(term.x, term.y);
+                                term.refresh(term.buffer.y, term.buffer.y);
```

Comment:
xterm uses `term.buffer.y` for both values when it calls refresh from showCursor(), so making this consistent

Reply:
Also I verified in xterm's typescript that term.buffer.y is public